api/provider/aws/common.go

```diff
@@ -243,3 +243,47 @@ func listClusterTaskARNs(ecsapi ecsiface.ECSAPI, clusterName, startedBy, status
 
 	return taskARNs, nil
 }
+
+func listContainerInstances(ecsapi ecsiface.ECSAPI, clusterName string) ([]string, error) {
+	containerInstanceARNs := []string{}
+	fn := func(output *ecs.ListContainerInstancesOutput, lastPage bool) bool {
+		for _, c := range output.ContainerInstanceArns {
```

Comment:
Do we need to filter by the state of the container instances? For example, will this list container instances that aren't actively running?

Reply:
Agreed - although this method isn't being used any longer, the same request being made from `environment_scaler.go` could would want to specify the `Status` property of the `ListContainerInstancesInput` (where status == ACTIVE)